packages/financial-templates-lib/src/price-feed/DefaultPriceFeedConfigs.js

```diff
@@ -361,6 +361,20 @@ const defaultConfigs = {
       { type: "cryptowatch", exchange: "gateio", pair: "amplusdt" },
       { type: "cryptowatch", exchange: "bitfinex", pair: "amplusd" }
     ]
+  },
+  // The following identifiers can be used to test how `CreatePriceFeed` interacts with this
+  // default price feed config.
+  TEST8DECIMALS: {
+    type: "test"
+  },
+  TEST6DECIMALS: {
+    type: "test"
+  },
+  TEST18DECIMALS: {
+    type: "test"
+  },
+  INVALID: {
```

Comment:
These are used by the CreatePriceFeed file to map between test identifiers and the Mock PriceFeeds